components/product/ProductInfo.js

```diff
@@ -10,15 +13,19 @@ import SpecialNotice from './SpecialNotice';
 function ProductInfo({ product }) {
   const { name, id, category, points, customerCost } = product;
   return (
-    <View style={{ width: 300 }}>
+    <ProductInfoContainer>
       <Title>{name}</Title>
-      <Subtitle>Category: {category}</Subtitle>
-      <View style={{ flexDirection: 'row' }}>
-        <Subtitle>${customerCost} each</Subtitle>
-        <Subtitle>Points Earned: {points}</Subtitle>
-      </View>
-      <SpecialNotice />
-    </View>
+      <ProductInfoCaptionContainer>
+        <Caption>${customerCost} each</Caption>
+        <Caption>Points Earned: {points}</Caption>
+      </ProductInfoCaptionContainer>
+      <ProductNoticeContainer>
+        <Body>
+          Note: not all products listed are available at every store. Please
```

Comment:
smol thing: capitalize the "n" in "not"? :0

Reply:
I couldn't find the exact prototype for this in figma but correct me if I'm wrong

Reply:
iirc it's in the Figma from last semester, was mistakenly removed but should be added back soon :o !